vegbilder-client/src/components/ImagePointsLayersWrapper/ImagePointsLayersWrapper.tsx

```diff
@@ -11,14 +11,14 @@ const ImagePointLayersWrapper = () => {
   const year = useRecoilValue(currentYearState);
 
   const renderImagePointsLayer = () => {
-    if (zoom > 14) {
+    if (zoom > 14 && year !== 'Nyeste') {
       return <ImagePointsLayer shouldUseMapBoundsAsTargetBbox={true} />;
-    } else if (zoom > 9) {
+    } else if (year === 'Nyeste' || zoom > 9) {
       return (
         <WMSTileLayer
           url="https://www.vegvesen.no/kart/ogc/vegbilder_1_0/ows"
           attribution="<a href='https://www.vegvesen.no/'>Statens vegvesen</a>"
-          layers={`Vegbilder_${year}`}
+          layers={year === 'Nyeste' ? `Vegbilder_${2020}` : `Vegbilder_${year}`} // det skal komme et annet kartlag her for nyeste (nå brukes 2020 som placeholder)
```

Comment:
Kanskje disse to kan trekkes ut som en funksjon som returnerer både  e.g. `Vegbilder_2020` og `Vegbilder_oversikt_2020`? :)

Reply:
Kan også bytte `Vegbilder_${2020}` med bare `Vegbilder_2020`